ARGO/src/CollisionSystem.cpp

```diff
@@ -82,6 +83,65 @@ bool CollisionSystem::circleToAABBCollision(Entity* t_entityCircle, Entity* t_en
 	return (deltaX * deltaX + deltaY * deltaY) < (radius * radius);
 }
 
+bool CollisionSystem::edgeOfTheWorldCircle(Entity& t_entity)
+{
+	TransformComponent* position = static_cast<TransformComponent*>(t_entity.getComponent(ComponentType::Transform));
+	int radius = static_cast<ColliderCircleComponent*>(t_entity.getComponent(ComponentType::ColliderCircle))->getRadius();
+	if (position->getPos().x - radius < 0)
+	{
+		Tag tag = static_cast<TagComponent*>(t_entity.getComponent(ComponentType::Tag))->getTag();
```

Comment:
You have code repeated in each of the if statements. If you can cleanup a bit it would be nice. If not thats fine

Reply:
I agree, this could be cleaned up or put in a function maybe

Reply:
this